lib/broccoli/fastboot-app-factory-plugin.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-env node */
+'use strict';
+
+const path = require('path');
+const fs = require('fs');
+const Plugin = require('broccoli-plugin');
+const fastbootAppFactoryModule = require('../utilities/fastboot-app-factory-module');
+
+module.exports = class FastBootAppFactory extends Plugin {
+  constructor(inputNode, options) {
+    super([inputNode], {
+      annotation: 'Generate: FastBoot app-factory module',
```

Comment:
can we update this annotation to include the exact filename?

`EmberCLIFastboot: write app-factory.js to output`